tests/integration/IntegrationTest.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const tape = require('tape')
+const http = require('http')
+const supertest = require('supertest')
+const {createTestEndpoint, removeTestEndpoint} = require('../testUtils')
+
+// Create an endpoint for the tests
+createTestEndpoint()
```

Comment:
Lets move this into the test setup instead of in the test module itself. It could then be updated in future to load a different test endpoint per test suite